plugins/package-managers/python/src/funTest/assets/projects/synthetic/poetry/poetry.lock

```diff
@@ -1,53 +1,72 @@
+# This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
+
 [[package]]
 name = "appdirs"
-version = "1.4.3"
+version = "1.4.4"
```

Comment:
Unrelated to this commit, but since you were investigating things, do you know why most of the packages in this lock file do not show up in the ORT dependency tree at all?

Reply:
> Unrelated to this commit, but since you were investigating things, do you know why most of the packages in this lock file do not show up in the ORT dependency tree at all?

I remind touching this thing and I have some guess, I'll investigate this next week. It should be relatively easy now, as I'm into it currently.